data/mods/Xedra_Evolved/field_type.json

```diff
@@ -136,5 +136,43 @@
     "priority": -1,
     "half_life": "20 minutes",
     "phase": "gas"
+  },
+  {
+    "type": "field_type",
+    "id": "fd_ginseng_scream",
+    "immune_mtypes": [ "mon_ginseng_protector" ],
+    "priority": 2,
+    "intensity_levels":
+    [
+      { "name": "ginseng_scream",
+        "sym": "^",
+        "color":  "brown",
+        "dangerous": true,
+        "move_cost": 120,
+        "effects":  
+        [
+          {
+            "effect_id": "warbled",
+            "min_duration": "1 minutes",
+            "max_duration": "72 hours", // Effect duration randomized between min and max duration
+            "intensity": 1,
+            "body_part": "head",
+            "immune_inside_vehicle": true,
+            "message": "You're vision shakes as the air warbles!",
```

Comment:
Unless it's intended to be "You are vision", this "You're vision" should be "Your vision" instead